packages/labs/virtualizer/src/support/resize-observer-errors.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {
+  interceptMethod,
+  MethodInterceptorTeardown,
+} from './method-interception.js';
+
+/**
+ * In a testing environment with a setup/before and teardown/after callback pattern, this function
+ * can be used to as a succinct declaration to ignore these errors in tests.
```

Comment:
is this more grammatically correct?
```suggestion
 * can be used as a succinct declaration to ignore these errors in tests.
```

Reply:
Fix applied!